pkg/apis/operator/v1alpha1/types_garden.go

```diff
@@ -127,6 +127,11 @@ type DNSDomain struct {
 
 // RuntimeNetworking defines the networking configuration of the runtime cluster.
 type RuntimeNetworking struct {
+	// IPFamilies specifies the IP protocol versions to use for the runtime cluster's networking. This field is
+	// immutable.
+	// Defaults to ["IPv4"].
+	// +optional
+	IPFamilies []gardencorev1beta1.IPFamily `json:"ipFamilies,omitempty"`
```

Comment:
Do we really need this field in the API? We should be able to deduce it from the fields below, i.e. nodes, pods and service. In general Kubernetes terms, the IP families play a role for defining a primary IP family, but we do not have this in the Garden runtime cluster as far as I know. It is analogous to other APIs, though.
WDYT?